rust/main/agents/relayer/src/relayer.rs

```diff
@@ -360,8 +350,16 @@ impl BaseAgent for Relayer {
             .await
             .into_iter()
             .collect::<HashMap<_, _>>();
+
+        let mut destination_chains = HashMap::new();
         for (destination, dest_mailbox) in mailboxes.iter() {
-            let destination_chain_setup = core.settings.chain_setup(destination).unwrap().clone();
+            let destination_chain_setup = match core.settings.chain_setup(destination) {
+                Ok(setup) => setup.clone(),
+                Err(err) => {
+                    tracing::error!(?destination, ?err, "chain_setup() failed");
```

Comment:
```suggestion
                    tracing::error!(?destination, ?err, "Destination chain setup failed");
```
nit: just a bit nicer / more readable than a reference to a function